backend/resolvers/types/userType.js

```diff
@@ -1,7 +1,18 @@
 export default {
   User: {
     // Judges (and Admins) have shows
-    shows (user) {
+    shows (user, args) {
```

Comment:
err.... I think this is put in the wrong place? Probably should be in `backend/resolvers/queries/showQuery.js`?